tq/basic_download.go

```diff
@@ -88,10 +88,12 @@ func (a *basicDownloadAdapter) download(t *Transfer, cb ProgressCallback, authOk
 		defer dlFile.Close()
 	}
 
-	rel, err := t.Actions.Get("download")
+	rel, err := t.Rel("download")
 	if err != nil {
 		return err
-		// return errors.New("Object not found on the server.")
+	}
+	if rel == nil {
+		return errors.New("Object not found on the server.")
```

Comment:
Good call, just pushed up 452bb6fb7742ec6d2e7b4d6183eea5b9815e0c56 to address. The errors are printed as-is, so before this would have just said "Object not found on the server" and then given up.